lib/modules/platform/github/common.ts

```diff
@@ -17,6 +17,7 @@ export function coerceRestPr(pr: GhRestPr): GhPr {
     state:
       pr.state === 'closed' && is.string(pr.merged_at) ? 'merged' : pr.state,
     bodyStruct,
+    body: pr.body,
```

Comment:
see other comment

Reply:
Sounds like a problem.. Any proposed solution for that problem?

At the moment I would probably use `etag` to know whether the PR has changed. Means when getting the PR (from the cache) make the call with the provided etag and serve from cache on 304.

Reply:
@viceice Any objection to use etag to resolve the caching problem? Or do you have any other suggestion?

Reply:
@rarkins I'm not sure how to proceed here. 🤔

Reply:
I don‘t know all the details about the integration into github (other systems). I would assume that events on pr updates are also handled by events. E.g. Rebase checkbox. Shouldn‘t they also update the cache?

Reply:
Renovate stores and compares a "hash" of the body content because storing the entire PR body is not efficient or necessary. This approach should still normally work for you to determine if the PR needs changing by comparing just the Renovate content.

In this new case though you need the full/raw PR body _once_ you decide it needs updating. We might need to add a new API or perhaps param to `getPr()` which fetches the full/raw PR body and returns it (even if it costs an extra API call).

Reply:
@rarkins, This PR takes care of some hashing, maybe something is missing.
Just to understand the flow. When a user updates the body, e.g., clicks the checkbox, renovate is triggered IMO.
I have seen some worker logic but I couldn't figure out how it gets the information about the checked options e.g., '<!-- rebase-check -->`. It seems always to use the cached version.
My assumption at the moment is that whenever a user updates the PR (e.g., body), renovate has to fetch again the PR to extract the check state of `rebase-check` and with that the cache is updated. The full body is also stored with the changes added in this PR.

If my thinking is correct, this PR is good to go IMO. If not, I would need some input or proposals to address the caching problem. As mention, one could use `etag` which is kind of expensive to ALWAYS get the latest body.

Reply:
we don't want to store the full body inside cache because of size.